subsys/bluetooth/controller/Kconfig

```diff
@@ -95,6 +95,13 @@ config BLUETOOTH_CONTROLLER_RX_PRIO_STACK_SIZE
 	int
 	default 448
 
+config BLUETOOTH_CONTROLLER_RX_PRIO
+	# Hidden option for Controller's Co-Operative high priority Rx thread
+	# priority.
+	int
+	depends on BLUETOOTH_HCI_HOST || BLUETOOTH_RECV_IS_RX_THREAD
```

Comment:
Does this really require any depends at all? Isn't it needed only if BLUETOOTH_CONTROLLER is selected, and that's already taken care of by this being inside controller/Kconfig